### PR TITLE
Update urls.py

### PR DESCRIPTION
**Pull request by F1ashhimself**
Remove duplicates

### CODE REVIEW

File: catalog/urls.py [hunk 1]
```diff
@@ -18,12 +18,6 @@
 ]
 
 
-urlpatterns += [
```
Comment (F1ashhimself) [inline catalog/urls.py]: The same lines you can find above.

Comment (hamishwillee) [inline catalog/urls.py]: I think you're incorrect - this change removes the code for a librarian to renew a book, which is "unique" code. Thoughts?

Comment (F1ashhimself) [inline catalog/urls.py]: This code is duplicated so removing this part will leave one more block about renew a book, you can verify it here https://github.com/mdn/django-locallibrary-tutorial/blob/150da77b13d24b0ef5c5ebfc7741830700c41c4c/catalog/urls.py looking at full file. Don't know why but this part was written twice in this file.

Comment (hamishwillee) [inline catalog/urls.py]: You're right. My apologies. Merged.

**Pull request merged**